.github/workflows/build.yml

```
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  release:
+    types: [ released ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    with:
+      upload_to_pypi: ${{ github.event_name == 'released'}}
```

```suggestion
      upload_to_pypi: ${{ github.event_name == 'release'}}
```
Based on the test here:
https://github.com/braingram/test/actions/runs/6300990973/job/17105025277#step:2:29
I think this should be 'release'. I wish I could point to docs somewhere that describes this but haven't been able to find any. Any good references @zacharyburnett ?

---

See https://github.com/spacetelescope/stdatamodels/pull/221#discussion_r1336045600 for a further improvement.